tools/do-destroy-rebuild-database

```diff
@@ -15,6 +15,7 @@ EOF
 
 sh "$(dirname "$0")/../scripts/setup/flush-memcached"
 
+./manage.py purge_queue message_sender
```

Comment:
@timabbott shouldn't stating the queue name (in this case message_sender) be sufficient to purge all the 10 (or fewer in low RAM case) `message_sender`'s?

Reply:
Yeah, but we also need to purge the missedmessage_email_notifications queue, etc.  You can get the list from running `./scripts/lib/queue_workers.py`, but for performance reasons, we'll want to purge them all from a single `manage.py` invocation; otherwise this will add like 10 seconds to the runtime of rebuilding the DB.

Reply:
I'd rather sacrifice the 10s to hotfix the problem rather than costing more to getting stuck in the errors.

Are the rest of the queues listed here (other than `message_sender`)?
https://github.com/zulip/zulip/blob/630ecf966904bd88d24da3091062d74c9acd4d4b/puppet/zulip/manifests/base.pp#L40-L57

Reply:
> I'd rather sacrifice the 10s to hotfix the problem rather than costing more to getting stuck in the errors.

I don't understand; there's no tradeoff between these.  We just need to add a `manage.py purge_queue --all` option that does them all in a loop, and have this command call that.  It can get the list from importing `zerver.worker.queue_processors.py`; there's a function that will return it.  It'll be almost no code and not have the 10s problem :)